src/dispatch/plugins/dispatch_slack/views.py

```diff
@@ -26,6 +26,7 @@
 from dispatch.decorators import background_task
 from dispatch.enums import Visibility
 from dispatch.event import service as event_service
+from dispatch.incident import enums as incident_enums
```

Comment:
It might be worth to be explicit about which Enums we're importing here in case the list grows in `dispatch.incident.enums`. We don't want to import more than needed. `IncidentStatus` a few lines below can be moved here too.